curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/check-for-all-or-none.english.md

```diff
@@ -32,13 +32,13 @@ Change the regex <code>favRegex</code> to match both the American English (favor
 ```yml
 tests:
   - text: Your regex should use the optional symbol, <code>?</code>.
-    testString: assert(favRegex.source.match(/\?/).length > 0);
+    testString: myRegex.lastIndex = 0; assert(favRegex.source.match(/\?/).length > 0);
   - text: Your regex should match <code>"favorite"</code>
-    testString: assert(favRegex.test("favorite"));
+    testString: myRegex.lastIndex = 0; assert(favRegex.test("favorite"));
   - text: Your regex should match <code>"favourite"</code>
-    testString: assert(favRegex.test("favourite"));
+    testString: myRegex.lastIndex = 0; assert(favRegex.test("favourite"));
   - text: Your regex should not match <code>"fav"</code>
-    testString: assert(!favRegex.test("fav"));
+    testString: myRegex.lastIndex = 0; assert(!favRegex.test("fav"));
```

Comment:
```suggestion
    testString: favRegex.lastIndex = 0; assert(!favRegex.test("fav"));
```

Reply:
Ooooh, I see, it's just that the variable is called favRegex in this lessson! Sorry about that. If I commit your requested changes that's okay?